build/build-modules-js/settings.json

```
@@ -539,47 +539,29 @@
         "dependencies": [],
         "licenseFilename": "LICENSE-MIT.txt"
       },
-      "@claviska/jquery-minicolors": {
-        "name": "minicolors",
+      "@ctrl/tinycolor": {
+        "name": "tinycolor",
         "js": {
-          "jquery.minicolors.js": "js/jquery-minicolors.js",
-          "jquery.minicolors.min.js": "js/jquery.minicolors.min.js"
-        },
-        "css": {
-          "jquery.minicolors.css": "css/jquery.minicolors.css"
-        },
-        "filesExtra": {
-          "jquery.minicolors.png": "css/jquery.minicolors.png"
+          "dist/index.js": "js/tinycolor.js"
         },
+        "dependencies": [],
         "provideAssets": [
           {
-            "name": "minicolors",
-            "type": "style",
-            "uri": "jquery.minicolors.css"
-          },
-          {
-            "name": "minicolors",
+            "name": "tinycolor",
             "type": "script",
-            "uri": "jquery.minicolors.min.js",
-            "dependencies": [
-              "jquery"
-            ],
+            "uri": "tinycolor.min.js",
             "attributes": {
               "defer": true
             }
           },
           {
-            "name": "minicolors",
+            "name": "tinycolor",
             "type": "preset",
             "dependencies": [
-              "minicolors#style",
-              "minicolors#script"
+              "tinycolor#script"
             ]
           }
         ],
-        "dependencies": [
-          "jquery"
-        ],
         "licenseFilename": "LICENSE.md"
```

This needs to be changed 

```suggestion
        "licenseFilename": "LICENSE"
```